utils/staking-miner/src/main.rs

```diff
@@ -296,29 +302,20 @@ struct Opt {
 async fn create_election_ext<T: EPM::Config, B: BlockT>(
```

Comment:
Maybe update this doc comment to `Build the `Ext` at `hash` with all the data of `ElectionProviderMultiPhase` and any `additional` pallets.